File: T7. DATA TYPES AND VARIABLES/P08.RefactorVolumeOfPyramid.cs
using System;

namespace P08.RefactorVolumeOfPyramid
{
    class Program
    {
        static void Main(string[] args)
        {
            double length, width, height, volume;

            Console.Write("Length: ");
            length = double.Parse(Console.ReadLine());

            Console.Write("Width: ");
            width = double.Parse(Console.ReadLine());

            Console.Write("Height: ");
            height = double.Parse(Console.ReadLine());

            volume = (length * width * height) / 3;

            Console.WriteLine("Pyramid Volume: {0:F2}", volume);

        }
    }
}

File: T7. DATA TYPES AND VARIABLES (Java)/P08_RefactorVolumeOfPyramid.java
import java.util.Scanner;

public class P08_RefactorVolumeOfPyramid {
    public static void main(String[] args) {

        Scanner in = new Scanner(System.in);

        double length, width, height, volume;

        System.out.print("Length: ");
        length = Double.parseDouble(in.nextLine());

        System.out.print("Width: ");
        width = Double.parseDouble(in.nextLine());

        System.out.print("Height: ");
        height = Double.parseDouble(in.nextLine());

        volume = (length * width * height) / 3;

        System.out.printf("Pyramid Volume: %.2f", volume);
    }
}
